index.bs

```diff
@@ -10518,126 +10522,140 @@ The characteristics of this property are as follows:
+                return <emu-val>undefined</emu-val>.
+            1.  If |attribute| is declared with a [{{PutForwards}}] extended attribute, then:
+                1.  Let |Q| be [=?=] [=Get=](|O|, |id|).
+                1.  If [=Type=](|Q|) is not Object, then [=ECMAScript/throw=] a
+                    <emu-val>TypeError</emu-val>.
+                1.  Let |forwardId| be the identifier argument of the [{{PutForwards}}] extended
+                    attribute.
+                1.  Perform [=?=] [=Set=](|Q|, |forwardId|, |V|).
                 1.  Return <emu-val>undefined</emu-val>.
-        1.  Let |idlValue| be an IDL value determined as follows:
-            *   If the type of the attribute is an [=enumeration=], then:
-                1.  Let |S| be the result of calling [=ToString=](|V|).
-                1.  If |S| is not one of the [=enumeration values|enumeration’s values=], then return <emu-val>undefined</emu-val>.
-                1.  The value of |idlValue| is the enumeration value equal to |S|.
-            *   Otherwise, the type of the attribute is not an [=enumeration=].
-                The value of |idlValue| is the result of [=converted to an IDL value|converting=]
-                |V| to an IDL value.
-        1.  If the attribute is a regular attribute, then perform the actions listed in the description of the attribute that occur when setting,
-            with |O| as the object and |idlValue| as the value.
-        1.  Otherwise, the attribute is a [=static attribute=].
-            Perform the actions listed in the description of the attribute that occur when setting with |idlValue| as the value.
-        1.  Return <emu-val>undefined</emu-val>.
-    </div>
+        1.  Let |idlValue| be determined as follows:
 
-    The value of the <emu-val>Function</emu-val> object’s “length”
-    property is the <emu-val>Number</emu-val> value <emu-val>1</emu-val>.
+            <dl class="switch">
+                <dt>|attribute|'s type is an [=enumeration=]</dt>
+                <dd>
+                    1.  Let |S| be [=?=] [=ToString=](|V|).
+                    1.  If |S| is not one of the [=enumeration values|enumeration’s values=], then
+                        return <emu-val>undefined</emu-val>.
+                    1.  Otherwise, |idlValue| is the enumeration value equal to |S|.
+                </dd>
+
+                <dt>Otherwise</dt>
+                <dd>
+                    |idlValue| is the result of [=converted to an IDL value|converting=] |V| to an
+                    IDL value of |attribute|'s type.
+                </dd>
+            </dl>
 
-    The value of the <emu-val>Function</emu-val> object’s “name”
-    property is a <emu-val>String</emu-val> whose value is the
-    concatenation of “set ” and the identifier of the attribute.
+        1.  Perform the actions listed in the description of |attribute| that occur on setting, on
+            |O| if |O| is not <emu-val>null</emu-val>.
```

Comment:
I still think this is pretty clear as-is, but we could do a different version perhaps as a follow-up if you have a proposal.